src/actions/actionTypes.js

```diff
@@ -0,0 +1,6 @@
+//Review Actions
+export const FETCH_REVIEWS = "FETCH_REVIEWS";
```

Comment:
A cool new thing to do with actions is to make them `Symbol`s - 

if we don't know about symbols

```javascript
const ACTION_A = Symbol('a');
const ACTION_A_2 = Symbol('a');

ACTION_A === ACTION_A_2; // false
```